### PR TITLE
Added Rules Page

### PR DESCRIPTION
**Pull request by nareshpoonia**
Added Rules Page

### CODE REVIEW

File: styles/index.css [hunk 1]
```diff
@@ -13,3 +13,9 @@
   width: 40rem;
   height: 40rem;
 }
+
+.card-container {
+  max-width: 400rem;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.2), 0 4px 16px 0 rgba(0, 0, 0, 0.2);
```
Comment (RakshithBhat03) [inline styles/index.css]: You can use a variable here if you want.

**Review by omgaikwad — APPROVED** [review]
Code looks clean, you can merge!

**Review by RakshithBhat03 — APPROVED** [review]
Everything looks good. The code is clean and well structured. Good to merge

File: styles/index.css [hunk 2]
```diff
@@ -13,3 +13,9 @@
   width: 40rem;
   height: 40rem;
 }
+
+.card-container {
+  max-width: 400rem;
```
Comment (Hardiegogo) [inline styles/index.css]: Are you sure about setting max-width to 400rem?

**Comment by Hardiegogo** [comment]
image

Try using `justify-content: center` here.

File: styles/index.css (hunk 1 above)
Comment (nareshpoonia) [reply styles/index.css]: Thanks for letting me know, variable used

File: styles/index.css (hunk 2 above)
Comment (nareshpoonia) [reply styles/index.css]: Thanks for pointing it out, it was a mistake, i was trying to check something, changed back to 50 rem now

**Pull request merged**